docs/install/install_instructions.md

```diff
@@ -116,22 +118,207 @@ pip3 install neuron
+
+'''
+#ps = h.PlotShape(False)
+#ps.variable("v")
+#ps.plot(pyplot, cmap=cm.cool)
+#pyplot.show()
+'''
+ps = h.PlotShape(False)
+ps.variable("v")
+ps.plot(plotly).show()
+```
+
+Inside the folder that your downloaded "c91662.ses" and saved "example_plotly_matplotlib.py" are, open a terminal and do:
+ 
+Open a new terminal and try: 
+```bash
+python -i example_plotly_matplotlib.py
+```
+ 
+---------------------- 
+ 
+### Common problems: 
+ 
+#### Incompatibility neuron and python version. See steps 1, 4 and 5. 
+ 
+The python version incompatibility with neuron version.
+
+---------------------- 
+```
+C:\Users\User>python3 
+Python 3.12.4 (tags/v3.12.4:8e8a4ba, Jun  6 2024, 19:30:16) [MSC v.1940 64 bit (AMD64)] on win32 
+Type "help", "copyright", "credits" or "license" for more information. 
+>>> import neuron 
+Traceback (most recent call last): 
+  File "c:\nrn\lib\python\neuron\__init__.py", line 135, in <module> 
+    from . import hoc 
+ImportError: cannot import name 'hoc' from partially initialized module 'neuron' (most likely due to a circular import) (c:\nrn\lib\python\neuron\__init__.py) 
+  
+During handling of the above exception, another exception occurred: 
+  
+Traceback (most recent call last): 
+  File "<stdin>", line 1, in <module> 
+  File "c:\nrn\lib\python\neuron\__init__.py", line 137, in <module> 
+    import neuron.hoc 
+ModuleNotFoundError: No module named 'neuron.hoc' 
+>>> 
+```
+
+---------------------- 
+
+#### "PATH" problems. See step 2. 
+ 
+A PATH problem that I'm not sure how to reproduce all error possibilities.  But it seems to be correlated with the python installation process (from Windows Microsoft Store (as an app) or by Python Webpage and/or Conda) and/or neuron/python order (I've stalled neuron before python and didn’t get this problem) 
+ 
+--------- Example 1 ------------- 
+```
+<stdin>:149: SyntaxWarning: invalid escape sequence '\.' 
+<stdin>:197: SyntaxWarning: invalid escape sequence '\.' 
+cygcheck: could not find 'C:/Users/User/AppData/Local/Microsoft/WindowsApps/PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0/python.exe' 
+ ```
+----------- End Example 1 ----------- 
+ 
+--------- Example 2 ------------- 
+```
+WARNING: The script f2py.exe is installed in 'C:\Users\User\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\Scripts' which is not on PATH. 
+  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location. 
+  WARNING: The scripts fonttools.exe, pyftmerge.exe, pyftsubset.exe and ttx.exe are installed in 'C:\Users\User\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\Scripts' which is not on PATH. 
+``` 
+----------- End Example 2 ----------- 
+ 
+ 
+If you face any of the issues above, it is due to a PATH missing line. You need to introduce the correct python path way at path compartment.
```

Comment:
I don't understand this wording, but you can set the PATH graphically through the Windows settings.